### ISSUE TITLE
[FR] Report not-FQN classes in @covers and @uses annotations

### ISSUE THREAD

**Issue by downace** [kalessil/phpinspectionsea#1617]
Description (screenshot):
--

PHPUnit requires FQN in `@covers` and `@coversDefaultClass` (at least <=8.5 versions).

It would be nice to see these errors in editor, like when method name is misspelled:

![Screenshot](https://i.ibb.co/T8fmgPt/image.png)

Example
--

`Foo.php`
```php
<?php
namespace App\Model;

class Foo {
    public function someMethod(): int {
        return 3;
    }
}
```

`FooTest.php`
```php
<?php
namespace App\Model; // <-- Same namespace, so no import required. But not for PHPUnit

class FooTest extends \PHPUnit\Framework\TestCase {
    /**
     * Here we need to specify a FQN: \App\Model\Foo::someMethod
     * @covers Foo::someMethod
     */
    public function testSomeMethod(): void {
        $foo = new Foo();
        $this->assertEquals(3, $foo->someMethod());
    }
}
```

Running tests with coverage outputs warning:
```
Trying to @cover or @use not existing method "Foo::someMethod".
```

But this is not reported in the editor:

![Screenshot](https://i.ibb.co/WHJ8RLG/image.png)
